Forge.TreeWalker/Forge.TreeWalker.csproj

```diff
@@ -49,63 +25,17 @@
     <PackageReference Include="GitVersionTask" Version="5.0.1">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="1.3.2" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="1.3.2" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="1.3.2" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="1.3.2" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Scripting" Version="1.3.2" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Scripting.Common" Version="1.3.2" />
-    <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Scripting" Version="3.8.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.2.0" />
-    <PackageReference Include="System.Reflection.Metadata" Version="1.2.0">
-      <NoWarn>NU1605</NoWarn>
-    </PackageReference>
-    <PackageReference Include="System.IO" Version="4.3.0" />
-    <PackageReference Include="System.Reflection" Version="4.3.0" />
-    <PackageReference Include="System.Runtime" Version="4.3.1" />
   </ItemGroup>
   <ItemGroup>
     <None Include="..\LICENSE.txt" Pack="true" PackagePath="$(PackageLicenseFile)" />
   </ItemGroup>
-  <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Threading.Tasks" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Runtime.Serialization" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="contracts\ForgeTree.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="src\ActionContext.cs" />
-    <Compile Include="src\ActionDefinition.cs" />
-    <Compile Include="src\ActionResponse.cs" />
-    <Compile Include="src\BaseAction.cs" />
-    <Compile Include="src\ExpressionExecutor.cs" />
-    <Compile Include="src\ForgeActionAttribute.cs" />
-    <Compile Include="src\ForgeDictionary.cs" />
-    <Compile Include="src\ForgeExceptions.cs" />
-    <Compile Include="src\IForgeDictionary.cs" />
-    <Compile Include="src\ITreeSession.cs" />
-    <Compile Include="src\ITreeWalkerCallbacks.cs" />
-    <Compile Include="src\ITreeWalkerCallbacksV2.cs" />
-    <Compile Include="src\SubroutineAction.cs" />
-    <Compile Include="src\TreeNodeContext.cs" />
-    <Compile Include="src\TreeWalkerParameters.cs" />
-    <Compile Include="src\TreeWalkerSession.cs" />
```

Comment:
will the new SDK style auto include all .cs files?

Reply:
Yes! Most of the messy tags that used to be required are default behavior in the new csproj format.

---
In reply to: [584981315](https://github.com/microsoft/Forge/pull/40#discussion_r584981315) [](ancestors = 584981315)